ultravox/training/train.py

```diff
@@ -169,8 +169,8 @@ def main() -> None:
     # To make sure we can compare training and validation loss (e.g. for fine-tuning), we keep a special set
     # called "matchtrain" that uses the same data as the training set.
     val_sets = dict(
-        [("matchtrain", args.data_sets)]
-        + [(x, [x]) for x in args.val_sets]
+        # [("matchtrain", args.data_sets)]
+        [(x, [x]) for x in args.val_sets]
```

Comment:
It turns out we have some issues regarding validation and GPU stalling. Apparently we've been using the `train` set for validation by mistake.

When I fixed that I ran into some issues: the first one was fixed by changing the number of shards on HeySQuAD, but GS is still giving me issues in validation so I disabled `matchtrain` for now specially since even if I fixed GS, we might still run into issues as we change `--data-sets`, which are gonna be harder to suspect (e.g. you change train-sets and now finally your training is stuck during validation ...).

Reply:
So this is a temporary fix for now, and we will add matchtrain back later? Maybe create an issue for the GPU stalling problem during validation?

Reply:
Done.